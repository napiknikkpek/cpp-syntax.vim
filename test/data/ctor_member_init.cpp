A::A() : Base{}, member1{}, member2() {}
