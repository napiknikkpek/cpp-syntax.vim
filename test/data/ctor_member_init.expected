t..f.....tttt....xxxxxxx....xxxxxxx.....
